<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputWidget</class>
 <widget class="QWidget" name="OutputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLineEdit" name="outputEdit">
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Output file...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="videoSelectionWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QRadioButton" name="noVideoButton">
            <property name="text">
             <string>No Video</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="videoCopyButton">
            <property name="text">
             <string>Copy Stream</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="videoTranscodeButton">
            <property name="text">
             <string>Transcode</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="videoTranscodeWidget" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="resizeButton">
            <property name="text">
             <string>Resize</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="aspectRatioLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>1.78:1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="widthLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="videoWidthButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>1920</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="heightLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="videoHeightButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>1080</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="frameRateButton">
            <property name="text">
             <string>Frame Rate</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="frameRateBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>5</number>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Custom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>23.976 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>29.97 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>59.94 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>60 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>120 fps</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="frameRateEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> fps</string>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>24.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Codec</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="videoCodecsBox">
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Bitrate</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="videoBitRateEdit">
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> Mbps</string>
            </property>
            <property name="maximum">
             <double>300.000000000000000</double>
            </property>
            <property name="value">
             <double>24.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Quality</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSlider" name="videoQualitySlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>75</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="qualityLabel">
            <property name="text">
             <string>Very good</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Audio</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="audioSelectionWidget" native="true">
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QRadioButton" name="noAudioButton">
            <property name="text">
             <string>No Audio</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="audioCopyButton">
            <property name="text">
             <string>Copy Stream</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="audioTranscodeButton">
            <property name="text">
             <string>Transcode</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="audioTranscodeWidget" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <widget class="QCheckBox" name="samplingButton">
            <property name="text">
             <string>Sampling</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="samplingBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Codec</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="audioCodecsBox">
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Bitrate</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="audioBitRateEdit">
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> kbps</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>320</number>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="audioQualityLabel">
            <property name="text">
             <string>(Nearly) Lossless</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSlider" name="audioQualitySlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Quality</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
